pyccel/parser/syntax/headers.py

```diff
@@ -96,6 +123,7 @@ def expr(self):
         d_var['allocatable'] = len(trailer)>0
         d_var['is_pointer'] = False
         d_var['precision']  = precision
+        d_var['isfunc'] = False
```

Comment:
All the other keys use underscores to separate words (ie `is_func`). I think it would be good to keep the same convention here